webapp/src/components/App/Tasks/JobTableView.tsx

```diff
@@ -84,16 +90,22 @@ function JobTableView(props: Props) {
         [
           filterRunningStatus && (({ status }: TaskView) => status === "running"),
           filterType && (({ type }: TaskView) => type === filterType),
+          userFilterValue && (({ userName }: TaskView) => userName?.includes(userFilterValue)),
```

Comment:
` userFilterValue && (({ userName }: TaskView) => isSearchMatching(userFilterValue, [userName]),`

Allows to be case insensitive and support username with accents.